.github/workflows/test.yml

```diff
@@ -744,7 +744,7 @@ jobs:
 
       - name: Test packages/client
         if: ${{ contains(needs.detect_jobs_to_run.outputs.jobs, '-all-') || contains(needs.detect_jobs_to_run.outputs.jobs, '-client-') }}
-        run: pnpm run test:functional:code && pnpm run test-notypes --testTimeout=40000
+        run: pnpm run test:functional:code && pnpm run test:memory && pnpm run test-notypes --testTimeout=40000
```

Comment:
Should we run it on buildkite as well? What are our rules on running stuff in GH actions only vs GH actions + Buildkite?

Reply:
The historical rule has been that everything that runs on GH also runs on BK. So I'd say yes by default, and since BK is our publish pipeline it also runs the tests before the release. Probably not ideal to maintain both? Let's discuss that some day.